package/MDAnalysis/core/accessors.py

```diff
@@ -63,10 +63,14 @@
    :members:
 
 """
+from __future__ import absolute_import
```

Comment:
What does this accomplish? According to the table in the standard library docs here: https://docs.python.org/3/library/__future__.html

`absolute_import` has been mandatory since Python `3.0`, and we have a higher minimum version of Python than that, right?

Reply:
Yes , I'll remove that part .